### PR TITLE
Tooltip specs

### PR DESCRIPTION
**Pull request by giladgrush**

### CODE REVIEW

File: docs/tooltip/tooltip.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+| Name | Type | Default | Required | Description |
+| -- | -- | -- | -- | -- |
+| id | string | null | no | Unique identifier |
+| className | string | null | no | Space separated list of the classes of the element |
+| style | Object | null | no | CSS styling declarations to be applied to the element |
+| children | React.ReactNode | null | no | The elements to render in the tooltip |
+| open | boolean | false | no | Whether the tooltip is shown or hidden |
+| onOpenStateChange | function | NOOP | no | Triggered when the popup is opened/closed |
+| anchor | Element | none | Yes | The element to be used as an anchor for the tooltip (will open next to it). |
+| maxHeight | number | 0 | no | The max height in pixels of the popup. If set to 0, the property will be ignored and receive height from its content |
```
Comment (tobich) [inline docs/tooltip/tooltip.md]: Shouldn't this be set in the styling via CSS?

File: docs/tooltip/tooltip.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+Esc key closes the tooltip. Not sure how it should be implemented here.
+
+### Mouse
+
+See triggers.
+
+### Touch
+
+None
```
Comment (tobich) [inline docs/tooltip/tooltip.md]: Maybe longpress should trigger as well?

File: docs/tooltip/tooltip.md (hunk 1 above)
Comment (giladgrush) [reply docs/tooltip/tooltip.md]: You can set it via CSS, however need it for TalmiJS...

File: docs/tooltip/tooltip.md (hunk 2 above)
Comment (giladgrush) [reply docs/tooltip/tooltip.md]: Not at this moment

**Pull request merged**